src/status_im/contexts/wallet/collectible/view.cljs

```diff
@@ -95,9 +97,26 @@
                            :picture     preview-url}}
          [rn/view {:style style/container}
           [rn/view {:style style/preview-container}
-           [rn/image
-            {:source preview-url
-             :style  style/preview}]]
+           [rn/touchable-opacity
+            {:active-opacity 1
+             :on-press       #(rf/dispatch [:lightbox/navigate-to-lightbox
+                                            token-id
+                                            {:images [{:image        preview-url
+                                                       :image-width  300 ; collectibles don't have
+                                                                         ; width/height but we need
+                                                                         ; to pass something
+                                                       :image-height 300 ; without it animation
+                                                                         ; doesn't work smoothly and
+                                                                         ; :border-radius not
+                                                                         ; applied
```

Comment:
The values are used in zooming animations. Have you tested how does zooming look.

Instead of using an arbitrary value we could use the device's width as the image dimensions.

Reply:
No, I didn't find any problems with resizing. 
Good idea, will try this!

Reply:
I tried using device dimensions and that resulted in wrong scaling and strange animation